include/zim/writer/item.h

```diff
@@ -114,10 +114,16 @@ namespace zim
          * to store).
          * The returned index data must stay valid even after creator release
          * its reference to the item.
+         * This method will be called once by libzim if it is compiled with xapian
+         * (and is configured to index data).
          *
-         * This method will be called twice by libzim if it is compiled with xapian
-         * (and is configured to index data). You may return the same indexData.
-         * The default implementation will use a contentProvider to get the content to index.
+         * The returned IndexData will be used as source to index the item.
+         * If you don't what the item to be indexed, you can return a nullptr here
```

Comment:
what → want

Reply:
Done.
I was a bit hurry in my rebase and directly fix the commit instead of creating a fixup commit, sorry about that.